md/list-of-documents.md

```diff
@@ -1,9 +1,42 @@
+
 # How to manage a list of documents
 
+Manage a list of documents through a process means add / edit / remove documents from a multiple data of type _document_.
+
+## Exemple: Document validation process
+
+::: warning
+This tutorial assume that you are already familiar with Bonita. If not you might want to read the [Getting started](getting-started-tutorial.md) first.
+:::
+
+This exemple shows how to easily create a document validation process using Bonita Studio.  
+A document validation process is a process where a User should be able to first submit a list fo documents. Then, an other user can perform a review on the list submitted, and ask for some changes on the list. If the reviewer asks for it, then the submitter has to edit the list of documents by adding / editing / removing some documents of the list, according to the review.  
+
+Here is a preview of the process we are going to build:  
+![Validation process](images/documentValidationProcess.png)
+
+ 1. Create a process _documentValidation_, with two lanes: _Submitter lane_ and _Reviewer lane_. 
+ 2. Create also two actors: _Submitter actor_ and _Reviewer actor_. Each actor should be mapped to a different part of the organization (two different users for exemple). Set the _Submitter actor_ as initiator, and map each lane to its actor.
+ 3. Create a multiple document variable named _documents_, and a process variable of type boolean named _submissionValid_.
+ 4. Select the pool and create a contract for the data _documents_.  ⚠️ Make sure to use the button **Add from data** to create your contract, else the contract input is not bounded to the data, you will have to code many things by yourself.
+ 5. Create the instantiation form using the Studio shortcut. Rename the form into _initDocuments_, and then go back to the Studio.
+ 6. Select the task _Review_. The objective of this task is to display the documents submitted to the reviewer. He should then be able to validate or not the submission, depending on the documents. To do so, we are going to use a small trick to gain some development time.
```

Comment:
:information_source: the trick shows us that there is a need for data management in the UID. Let's do it in the next release and come back on this tutorial to amend it.